lib/src/models/community_feed.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+import 'package:lightning_api/src/models/models.dart';
+
+part 'community_feed.g.dart';
+
+@JsonSerializable(explicitToJson: true)
+class CommunityFeed extends Equatable {
+  const CommunityFeed({
+    required this.posts,
+    required this.lastOpIndices,
+    required this.oldestQuery,
+  });
+
+  factory CommunityFeed.fromJson(Map<String, dynamic> json) {
+    return _$CommunityFeedFromJson(json);
+  }
+
+  @JsonKey(
+    fromJson: Authorperm.deserializeList,
+    toJson: Authorperm.serializeList,
+  )
+  final List<Authorperm> posts;
+  final Map<String, int> lastOpIndices;
```

Comment:
Right now, client doesn't need this information, I use same class to cache it though, so it is needed in cache rather than client, so I agree it doesn't need to be exposed in request. I can create a local struct for it and use that instead.